examples/profile/index.js

```diff
@@ -68,6 +68,7 @@ router.get('/profile', (req, res) => {
 
     res.render('pages/profile', {
       userId: activityDetails.getUserId(),
```

Comment:
Not essential here, but maybe this is a good opportunity to deprecate _userId_ and replace it with _rememberMeId_